sklearnex/linear_model/coordinate_descent.py

```diff
@@ -15,3 +15,12 @@
 # ===============================================================================
 
 from daal4py.sklearn.linear_model import ElasticNet, Lasso
+from onedal._device_offload import support_usm_ndarray
+
+ElasticNet.fit = support_usm_ndarray(queue_param=False)(ElasticNet.fit)
+ElasticNet.predict = support_usm_ndarray(queue_param=False)(ElasticNet.predict)
+ElasticNet.score = support_usm_ndarray(queue_param=False)(ElasticNet.score)
+
+Lasso.fit = support_usm_ndarray(queue_param=False)(Lasso.fit)
+Lasso.predict = support_usm_ndarray(queue_param=False)(Lasso.predict)
+Lasso.score = support_usm_ndarray(queue_param=False)(Lasso.score)

```

Comment:
```suggestion
from onedal._device_offload import support_usm_ndarray

class ElasticNet(daal4py_ElasticNet):
    fit = support_usm_ndarray(queue_param=False)(daal4py_ElasticNet.fit)
    predict = support_usm_ndarray(queue_param=False)(daal4py_ElasticNet.predict)
    score = support_usm_ndarray(queue_param=False)(daal4py_ElasticNet.score)

class Lasso(daal4py_Lasso):
    fit = support_usm_ndarray(queue_param=False)(daal4py_Lasso.fit)
    predict = support_usm_ndarray(queue_param=False)(daal4py_Lasso.predict)
    score = support_usm_ndarray(queue_param=False)(daal4py_Lasso.score)

```

Reply:
In this case I also should bring docstrings from daal4py, am I right?

Reply:
But I think this potentially could be an issues that here used only daal4py imported estimator. However, I should check that the methods here are indeed overridden

Reply:
This kind of decorations works fine.
It doesn't change current daal4py, sklearnex dispatch mapping, except moving `support_usm_ndarray` decoration into sklearnex level. See also: [support_usm_ndarray](https://github.com/intel/scikit-learn-intelex/pull/1832/files#r1649469866)